### DISCUSSION TITLE
I've implemented mouse RR

### DISCUSSION THREAD

**Discussion by godfetish** [OpenMultiBoxing/OpenMultiBoxing#3] [General]
https://github.com/godfetish/OMB_MouseRR

Thank you moorea for OMB! Between EMA, DB, and OMB I can ALMOST play how I used to play.

I've been tinkering around for a while with this idea and my latest iteration is released. I didn't submit a pull because I wasn't sure if mouse RR was OK in your opinion for OMB or WOB..
![2021-12-21 21_42_40-C__Users_johnk_Desktop_BuildKit-main](https://user-images.githubusercontent.com/6171211/147026302-ace9ef44-6030-40be-bd44-8763d8cd6fe5.png)
.

**Comment by godfetish** [comment]
![2021-12-21 21_44_34-](https://user-images.githubusercontent.com/6171211/147026552-e331fd5a-4b50-4fa4-a361-79b7c9254e96.png)

**Reply by aDwCarrazzone** [reply]
Do you still have this? :)

**Comment by mooreatv** [comment]
Hey I didn't see this until now somehow (github isn't very good at notifications)

You repo is either private or gone... 

@aDwCarrazzone did you see that OMB supports now full mouse left and right click broadcasting?
